components/clp-package-utils/clp_package_utils/general.py

```diff
@@ -241,6 +257,43 @@ def generate_container_config(clp_config: CLPConfig, clp_home: pathlib.Path):
     return container_clp_config, docker_mounts
 
 
+def dump_container_config(
+    clp_config: CLPConfig, container_clp_config: CLPConfig, container_name: str
```

Comment:
Can we swap `container_clp_config` and `clp_config`?

Reply:
technically, we don't need the `clp_config`. the only thing needed by this function is `clp_config.logs_directory` which can be an argument called `host_log_directory`.
I didn't write in this way because then `clp_config.logs_directory` need to be duplicated in all callers.
me know what you think

Reply:
Since `clp_config.logs_directory` and `container_clp_config.logs_directory` are linked, I figure it's safer to pass in `container_clp_config` and `clp_config` to lessen the risk that the caller passes in two random paths.